jaws-journey-deploy/orb.yml

```diff
@@ -237,6 +255,50 @@ commands:
             npx -p @semantic-release/changelog -p @semantic-release/git -p semantic-release semantic-release --debug=true --tag-format=\${version} --branches='master' --plugins='@semantic-release/commit-analyzer' '@semantic-release/release-notes-generator' '@semantic-release/changelog' '@semantic-release/git' '@semantic-release/github' --repositoryUrl="https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git"
           name: Run semantic versioning
 
+  tf:
+    parameters:
+      <<: *tf-common-parameters
+      command:
```

Comment:
could this be added to the tf-common-parameters?

Reply:
It would make it required in all the tf-apply tf-plan etc jobs. So it's just to ensure it's only required for the tf job

Reply:
hmm in that case might look at updating later as feels weird that we have 2 ways to do the same thing